src/components/shared/ButtonsLarge/index.tsx

```diff
@@ -0,0 +1,34 @@
+import classnames from 'classnames';
+
+export enum ButtonLargeVariants {
+  Primary = 'primary',
+  Outline = 'outline',
+}
+
+interface ButtonLargeProps {
+  variant: string;
```

Comment:
If you accept specific variants it'd be nice to declare that, not just allow any string.. you can do something like
```suggestion
  variant: 'MyExpectedVariant' | 'AnotherVariant';
```
and so on.. 
you can even make it your own type
`type VariantType = 'MyExpectedVariant' | 'AnotherVariant';`
and then
```suggestion
  variant: VariantType;
```